jobs/baml_us_functional_programmer.html

```diff
@@ -0,0 +1,47 @@
+---
+title: Software Developer
+company: Bank of America Merrill Lynch
+url: https://www.python.org/jobs/702/
+location: 250 Vesey St, New York City, New York, USA
+contract: permanent
+contact:
+    name: Robert Donnelly
+    email: robert.donnelly@baml.com
+    phone: +1 212 449 1135
+created: !!timestamp '2015-12-08' # Update this with today's date
+tags:
+- python
+- scala
+- coffeescript
+- web
+- finance
+- functional
+---
+
+# Job Description
+
+We are looking for Python developers to join our teams in our Downtown offices. We consider developers from all backgrounds
+and all levels of experience. Financial experience is desirable but not a requirment. We can offer flexible conditions
+and a stimulating, creative atmosphere. We have vacancies for both contractors and permanent employees.
+
+# Restrictions
+
+* No telecommuting, this is primarily an office based role.
+* No agencies please
+
+# Requirements
+
+Fluency in at least one of the programming languages we use (Python, Scala, JavaSceipt): That means in addition to knowing the
+core of a language, we expect knowledge of it's , it's types and standard library. Anything else you might bring to the job is a bonus.
```

Comment:
"it's" what?

"it's" is short for "it is". "its" is the possessive form of "it".